src/main/kotlin/org/jmailen/gradle/kotlinter/KotlinterPlugin.kt

```diff
@@ -37,25 +36,37 @@ class KotlinterPlugin : Plugin<Project> {
                 sourceResolver.applyToAll(project) { id, resolveSources ->
                     val lintTaskPerSourceSet = tasks.register("lintKotlin${id.capitalize()}", LintTask::class.java) { lintTask ->
                         lintTask.source(resolveSources)
-                        lintTask.ignoreFailures = kotlinterExtension.ignoreFailures
-                        lintTask.reports = kotlinterExtension.reporters.associate { reporter ->
-                            reporter to reportFile("$id-lint.${reporterFileExtension(reporter)}")
-                        }
-                        lintTask.ktLintParams = kotlinterExtension.toKtLintParams(editorConfigPath())
-                        lintTask.fileBatchSize = kotlinterExtension.fileBatchSize
+                        lintTask.ignoreFailures.set(provider { kotlinterExtension.ignoreFailures })
```

Comment:
I assume using this lifecycle user customized values on specific tasks will still take precedence and this documentation is still accurate?
https://github.com/jeremymailen/kotlinter-gradle#customizing-tasks

Reply:
It should as I had that in mind when writing the code, but I'll add a test for that 👍